soh/src/overlays/actors/ovl_En_Mb/z_en_mb.c

```diff
@@ -507,6 +507,13 @@ void EnMb_SetupClubAttack(EnMb* this) {
     f32 frames = Animation_GetLastFrame(&gEnMbClubLiftClubAnim);
     s16 relYawFromPlayer;
 
+    // Rotate Club Moblin towards player in Enemy Randomizer because they're borderline
+    // useless otherwise in most scenarios.
```

Comment:
```suggestion
    // Rotate Club Moblin towards player in Enemy Randomizer because they're
    // borderline useless otherwise in most scenarios.
```

nitpick: when i first read this i didn't realize that it was saying "borderline useless," so i thought it was 2 separate things
* they're borderline (i had no idea what that was supposed to mean)
* useless in most scenarios (also was very confusing)